source/github-helpers/github-url.test.ts

```diff
@@ -1,7 +1,7 @@
 import test from 'ava';
 
-import './fixtures/globals';
-import GitHubURL from '../source/github-helpers/github-url';
+import '../../fixtures/globals';
```

Comment:
Worth setting up a "path shortcut" in `tsconfig.json` (e.g. `~`) to avoid this?

Reply:
I'd rather not, Maybe there's a way to have AVA automatically load this as an env/setup file